runners/spark/src/main/java/org/apache/beam/runners/spark/SparkTransformOverrides.java

```diff
@@ -50,6 +53,18 @@ public static List<PTransformOverride> getDefaultOverrides(boolean streaming) {
               PTransformOverride.of(
                   PTransformMatchers.urnEqualTo(PTransformTranslation.SPLITTABLE_PROCESS_KEYED_URN),
                   new SplittableParDoNaiveBounded.OverrideFactory()));
+    } else {
+      // For streaming pipelines, this override is applied only when the PTransform has the same URN
```

Comment:
I understand this comment goes to the first item of "PTransformOverride.of("

Reply:
@Abacn 
Thanks for the review.
I'll move the comment above the first argument of `PTransformOverride.of`.